servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/FlushStrategyOnServerTest.java

```diff
@@ -112,19 +125,36 @@ public void flush(final ChannelHandlerContext ctx) {
             return succeeded(resp);
         };
         DefaultHttpExecutionContext httpExecutionContext =
-                new DefaultHttpExecutionContext(DEFAULT_ALLOCATOR, ioExecutor, executor, executionStrategy);
-        ReadOnlyHttpServerConfig config = new HttpServerConfig().asReadOnly();
+                new DefaultHttpExecutionContext(DEFAULT_ALLOCATOR, ioExecutor, executor, param.executionStrategy);
+
+        final ReadOnlyHttpServerConfig config = new HttpServerConfig().asReadOnly();
         serverConnection = initChannel(channel, httpExecutionContext, config,
-                new TcpServerChannelInitializer(config.tcpConfig()), service, true,
+                new TcpServerChannelInitializer(config.tcpConfig()) {
+                    @Override
+                    public void init(final Channel channel) {
+                        super.init(channel);
+                        channel.pipeline().addLast(new ChannelInboundHandlerAdapter() {
+                            @Override
+                            public void userEventTriggered(final ChannelHandlerContext ctx,
+                                                           final Object evt) throws Exception {
+                                if (evt == CloseHandler.ProtocolPayloadEndEvent.OUTBOUND) {
+                                    // Mute payload boundary events for this test with repeated read
+                                    return;
+                                }
+                                super.userEventTriggered(ctx, evt);
+                            }
+                        });
+                    }
+                }, service, true,
                 UNSUPPORTED_PROTOCOL_CLOSE_HANDLER)
                 .toFuture().get();
         serverConnection.process(true);
         headersFactory = DefaultHttpHeadersFactory.INSTANCE;
     }
 
     @Parameters(name = "{index}: strategy = {0}")
-    public static Collection<HttpExecutionStrategy> data() {
-        return asList(noOffloadsStrategy(), defaultStrategy(), customStrategyBuilder().offloadAll().build());
+    public static Param[][] data() {
```

Comment:
Is the use of `enum` here just a personal choice or there is a difference between the two approaches?

Reply:
Not personal choice, this improves tooling (IDE/gradle) support. With the existing approach you can't run/debug individual test-cases due to unstable name.

Reply:
ok, great. thanks